### PR TITLE
Fix ProjectSettings.save_custom documentation

### LINKED ISSUE #38152: ProjectSettings.save_custom returns error ERR_FILE_UNRECOGNIZED for any path
**Godot version:**
v3.2.2.rc.custom_build.36a30f681

**OS/device including version:**
Window 10

**Issue description:**
I'm trying to save ProjectSettings at runtime using save_custom to location "user://override.cfg", but this action returns error ERR_FILE_UNRECOGNIZED. Actually this method returns this error for any path. Is it allowed to call it at runtime or only in tool? I want to use ProjectSettings mechanism at runtime for storing some user configurations.

### PR DESCRIPTION
**Pull request by gvekan**
Fixes https://github.com/godotengine/godot/issues/38152

### CODE REVIEW

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -137,7 +137,7 @@
 			<argument index="0" name="file" type="String">
 			</argument>
 			<description>
-				Saves the configuration to a custom file.
+				Saves the configuration to a custom file. The file extension must be [code].godot[/code] or [code].binary[/code].
```
Comment (Calinou) [inline doc/classes/ProjectSettings.xml:140]:
```suggestion
				Saves the configuration to a custom file. The file extension must be [code].godot[/code] (text-based [ConfigFile] format) or [code].binary[/code] (binary format).
```

**Review by Calinou — APPROVED** [review]

**Comment by akien-mga** [comment]
Could you squash commits into one? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase).

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**